samples/simple/SimpleModule.hx

```diff
@@ -0,0 +1,28 @@
+#if !macro
+private typedef Import = haxe.macro.MacroType<[SimpleModule.build()]>; 
+#else
+
+class SimpleModule {
+
+	static var json = {
```

Comment:
I'm thinking we can simplify this block like so:

```suggestion
	static var json = haxe.Json.parse(sys.io.File.getContent("./webidl.json"));
```

Reply:
The GitHub auto-commit thing won't remove the 4 lines below, so we'll need to do that manually.